server/config/constants.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  DB_TYPE: {
+    MONGO: 'MONGO',
+    POSTGRES: 'POSTGRES',
+    EMPTY: null
```

Comment:
Also this should probably be DB_TYPES (plural), which also makes it less confusing when used with appConfig.DB_TYPE